fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -176,21 +180,37 @@
             .optionalEnd().toFormatter();
 
     private FHIRPersistence persistence = null;
+    private FHIRVersionParam fhirVersion = null;
 
     // Used for correlating requests within a bundle.
     private String bundleRequestCorrelationId = null;
 
-    private final FHIRValidator validator = FHIRValidator.validator(FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_VALIDATION_FAIL_FAST, Boolean.FALSE));
+    private final FHIRValidator validator = FHIRValidator.validator(FHIRConfigHelper.getBooleanProperty(PROPERTY_VALIDATION_FAIL_FAST, Boolean.FALSE));
 
+    /**
+     * Construct an instance with the passed FHIRPersistence and a FHIRVersion of 4.3.0
+     * @see #FHIRRestHelper(FHIRPersistence, FHIRVersion)
+     */
     public FHIRRestHelper(FHIRPersistence persistence) {
```

Comment:
should this be deprecated now?

Reply:
I think its fine as a convenience for contructing instances with a version that matches the "actual" implementation